### DISCUSSION TITLE
My custom domain does not work

### DISCUSSION THREAD

**Discussion by Symphonic3** [community/community#22328] [Other Feature Feedback, Questions, & Ideas]
<p>Basically i made a CNAME entry with my DNS provider that points <a href="http://games.audaxgames.net" rel="nofollow noopener">games.audaxgames.net</a> to <a href="http://symphonicyt.github.io" rel="nofollow noopener">symphonicyt.github.io</a>.</p>
<p>On my github repo, i made the custom domain <a href="http://games.audaxgames.net" rel="nofollow noopener">games.audaxgames.net</a>. after going to <a href="http://games.audaxgames.net/rollaball" class="inline-onebox-loading" rel="nofollow noopener">games.audaxgames.net/rollaball</a> (the repo) i get an error 404. -_- what am i doing wrong?</p>
<p>Edit: ok lol instead of <a href="http://games.audaxgames.net/rollaball" class="inline-onebox-loading" rel="nofollow noopener">games.audaxgames.net/rollaball</a> i just need to use <a href="http://games.audaxgames.net" rel="nofollow noopener">games.audaxgames.net</a>, so now im asking how to make it so the link is <a href="http://games.audaxgames.net/rollaball" class="inline-onebox-loading" rel="nofollow noopener">games.audaxgames.net/rollaball</a> instead.</p>

**Comment by mpboom** [comment]
<p><a href="https://www.audaxgames.net/" rel="nofollow noopener">https://www.audaxgames.net/</a> works fine for me.</p>

**Comment by ghost** [comment]
<p>Hey there <a class="mention" href="/symphonicyt">@symphonicyt</a>,</p>
<p>Welcome to the GitHub Community Forum, it’s great to have you here!</p>
<p>Alongside what <a class="mention" href="/mpboom">@mpboom</a> said, I’m also not having any issues either with accessing your site from <a href="https://www.audaxgames.net/" rel="nofollow noopener">https://www.audaxgames.net/</a> or <a href="https://games.audaxgames.net/" rel="nofollow noopener">https://games.audaxgames.net/</a>. If you’re still having issues, does clearing your browsers cookies and cache help?</p>
<p>In terms of <a href="https://games.audaxgames.net/rollaball/" rel="nofollow noopener">that specific page</a> too, it takes me to a Unity WebGL game - should it be going somewhere else?</p>

**Comment by Symphonic3** [comment]
<p>The root domain points to www which points to google site, i only used github pages for a subdomain.</p>

**Comment by Symphonic3** [comment, marked as answer]
<p>Yeah, so basically <a href="http://www.audaxgames.net" rel="nofollow noopener">www.audaxgames.net</a> is a google site and from a certain page there, i want to redirect to github, where i can upload my Unity WebGL games. I created a subdomain for that, <a href="https://games.audaxgames.net" rel="nofollow noopener">games.audaxgames.net</a>. The problem was that that subdomain pointed to a repo which had my game, therefore i could only display that one game, and the title was wrong. I ended up renaming the repo to “games” and withing that repo i created a directory named “rollaball”, so it now shows the game at <a href="https://games.audaxgames.net/rollaball" rel="nofollow noopener">games.audaxgames.net/rollaball</a>. <a href="http://games.audaxgames.net" rel="nofollow noopener">games.audaxgames.net</a> contains a simple HTML file which redirects to my google website at www. now. I think I’ve got it all sorted, thanks for helping anyways!</p>

**Comment by ghost** [comment]
<p>Awesome, I’m so glad you were able to get this sorted <a class="mention" href="/symphonicyt">@symphonicyt</a>!</p>
<p>If you need a hand with anything else, just let me know and I’ll come running to help :running_man:‍♂</p>
